aggregation/quantile/cm/options.go

```diff
@@ -62,51 +64,66 @@ func NewOptions() Options {
 	floatsPool := NewFloatsPool(defaultBuckets, nil)
 	floatsPool.Init()
 
-	return options{
+	return &options{
```

Comment:
More or less for consistency, since i'm also using a pointer for aggregator options. Passing pointers is cheaper than passing values, especially given how many fields the aggregator options have.